### ISSUE TITLE
Reader: posts in full-post view for sites you don't already follow do not have Follow button

### ISSUE THREAD

**Issue by jancavan** [Automattic/wp-calypso#4843]
#### Steps to reproduce
1. From a stream other than Followed Posts, click on a post title to bring up full-post view

#### What I expected
A Follow button on full-post view:

![screenshot 2016-04-18 16 04 59](https://cloud.githubusercontent.com/assets/4924246/14622522/50c476e4-057f-11e6-9ec4-e7568e1b040e.png)

When you're following a site, we already display the Following button:

![screenshot 2016-04-18 16 12 09](https://cloud.githubusercontent.com/assets/4924246/14622689/b9981cb0-0580-11e6-9721-5c290d232680.png)

#### What happened instead
Follow button is missing on full-post:

![screenshot-2016-04-18-16 05 43](https://cloud.githubusercontent.com/assets/4924246/14622677/a9b5fb96-0580-11e6-99e4-d823f2fc6305.jpg)

#### Browser / OS version
Chrome 49.0.2623  / Mac OS 10.11.3

Ht: @rickybanister

**Comment by lancewillett** [comment]
I can't repeat this one, testing with Safari 9.1 on OS X 10.11.4 and https://wpcalypso.wordpress.com — seems to be working as expected. Same with Chrome, testing with various users.

**Comment by jancavan** [comment]
@lancewillett when you click on the post title from any stream (that's not Followed Sites), it brings up the full-post view that doesn't display the Follow button when it should be.

**Comment by lancewillett** [comment]
Aha! I missed the important bit: the links are missing **only when you don't already following the site**.

**Comment by rickybanister** [comment]
Yes indeed, the place where it is actually important.

**Comment by v18** [comment]
This has been fixed in the new reader, closing as fixed.

Here's what I see in the full-post view for a site I am not following:

<img width="810" alt="screen shot 2016-08-13 at 5 39 44 pm" src="https://cloud.githubusercontent.com/assets/11487924/17645764/fad99782-617c-11e6-8886-79c37f8a1f3d.png">


The steps I took to get to this site:
1. Go to the reader
2. Pick a tag
3. Click on a post's title to see full-post view
